pyproject.toml

```diff
@@ -30,7 +30,8 @@ dependencies = [
     "pluggy>=1.0",
     "PyYAML>=4.2,<7.0",
     "rich>=12.0,<15.0",
-    "toml>=0.10.0",
+    "tomli>=1.1.0;python_version<'3.11'",  # tomllib is part of the standard library in Python 3.11+
+    "tomli-w",
```

Comment:
Why do you need the separate `tomli-w` dependency?

Reply:
`tomli` (and the `tomllib` included in the Python standard library from 3.11) are read-only, and `tomli-w` is the recommended writing library (same author).

Reply:
Got it! Shame they didn't put it in the same library 🥲